py/parse_and_notify.py

```diff
@@ -123,6 +122,18 @@ def build(cls, raw_content: Mapping[str, Any]) -> UpdatedTopic:
                     for entry in raw_content["missingKeys"] or []
                 ]
             )
+        if raw_content["replicaAssignments"]:
```

Comment:
you can see a test event with this change [here](https://app.datadoghq.com/event/explorer?query=source%3Atopicctl%20sentry_region%3Aben_test%20&cols=&event=AgAAAZCX2Xu4aCSjNQAAAAAAAAAYAAAAAEFaQ1gyWHU0QUFDdHV3LUJLTjg5YVdxdwAAACQAAAAAMDE5MDk3ZDktODc3MC00ZWY5LTkzMzEtODhhNmYxNGFjMDYx&messageDisplay=expanded-lg&options=&refresh_mode=sliding&sort=DESC&view=spans&from_ts=1720530878069&to_ts=1720534478069&live=true)